stdlib/contrib/RohanSreerama5/naiveBayesClassifier/naiveBayesClassifier.flux

```diff
@@ -1,11 +1,24 @@
-//Final working code as of August 11, 2020
-//Currently supports single field classification and binary data sets 
-//Please ensure Ruby is installed
+// Package naiveBayesClassifier provides an implementation of
+// a naive Bayes classifier.
+//
+// Currently supports single field classification and binary data sets.
+//
+// For information about demonstrating functions in this package, see the
+// [package README on GitHub](https://github.com/influxdata/flux/blob/master/stdlib/contrib/RohanSreerama5/naiveBayesClassifier/README.md).
 package naiveBayesClassifier
 
 
 import "system"
 
+
+// naiveBayes performs a naive Bayes classification.
+//
+// ## Parameters
+//
+// - myMeasurement: Measurement to use as training data.
+// - myField: Field to use as training data.
+// - myClass: Class to classify against.
+// - tables: Input data. Default is piped-forward data (`<-`).
 naiveBayes = (tables=<-, myClass, myField, myMeasurement) => {
```

Comment:
fluxdoc is having trouble handling `tables=<-`. I'll look into it 😈

Reply:
@nathaniel, are we omitting `tables` parameters from parameter lists?